account_move_batch_validate/wizard/move_marker.py

```diff
@@ -38,11 +38,16 @@ class AccountMoveMarker(orm.TransientModel):
             ('mark', 'Mark for posting'),
             ('unmark', 'Unmark for posting'),
         ], "Action", required=True),
-        'eta': fields.integer('Seconds to wait before starting the jobs')
+        'eta': fields.integer('Seconds to wait before starting the jobs'),
+        'journal_ids': fields.many2many('account.journal',
+                                        'account_post_journal_rel',
+                                        'wiz_id', 'journal_id', 'Journals',
+                                        required=True),
```

Comment:
According with Stéphane and concidering that this module replace the official menu for draft entries validation, I'll propose a solution to reajust this one from the official wizard.